projects/optic/src/commands/diff/upload-diff.ts

```diff
@@ -1,15 +1,22 @@
+import ora from 'ora';
 import { OpticCliConfig, VCS } from '../../config';
 import { OPTIC_URL_KEY } from '../../constants';
-import { getApiFromOpticUrl } from '../../utils/cloud-urls';
+import { getApiFromOpticUrl, getRunUrl } from '../../utils/cloud-urls';
 import { ParseResult } from '../../utils/spec-loaders';
 import { EMPTY_SPEC_ID, uploadRun, uploadSpec } from '../../utils/cloud-specs';
 import * as Git from '../../utils/git-utils';
+import { logger } from '../../logger';
 
 export async function uploadDiff(
   specs: { from: ParseResult; to: ParseResult },
   specResults: Parameters<typeof uploadRun>['1']['specResults'],
   config: OpticCliConfig
 ): Promise<{ runId: string; apiId: string; orgId: string } | null> {
+  const showSpinner = logger.getLevel() !== 5;
```

Comment:
Easiest way I could get this to respect the log level without trying to override the `ora` writable stream